### PR TITLE
Add example with CSS

### PR DESCRIPTION
**Pull request by OzzieOrca**
- Rename description field to category
- Add header class

Based on https://www.figma.com/file/quU7g6WkdQ0hDiSrtDnSVA/Content-Based-Filtering?node-id=2%3A0. I haven't worked on making it responsive yet.

### CODE REVIEW

**Comment by j2trumpet** [comment]
Should the branding submodule be incorporated here so the colors and font sizes and responsiveness remain consistent?

**Comment by OzzieOrca** [comment]
Idk. Someone (maybe you or Bill) is going to move this into AEM and you can do whatever you want. Idk if you even want my CSS. I initially wasn't going to write any CSS and let AEM/Wordpress style it. But I was asked to write some initial/example CSS and that'll be nice for sharing with the rest of the team as a prototype.

**Comment by j2trumpet** [comment]
Gotcha. Then perhaps we should leave it as-is in case it gets integrated before the new components styles with the branding submodule get launched.

File: example-with-css.css [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  text-transform: uppercase;
+  color: #ffcf07;
+}
+.cru-recommendations-component .cru-recommendations-component-title {
+  font-size: 18px;
+}
+.cru-recommendations-component
+  .cru-recommendations-component-card:hover
+  .cru-recommendations-component-title {
+  color: #5a5a5a;
```
Comment (j2trumpet) [inline example-with-css.css:52]: @OzzieOrca This isn't a branding color anymore and I haven't gotten direction from Eric about any color variations for hover effects, so let's swap this color change for `text-decoration: underline;`.

**Comment by wrandall22** [comment]
This will go live before the new branding goes live. I don't know if the Figma is using the new or old branding.

**Review by j2trumpet — APPROVED** [review]
I think the CSS could use some refinement once it's integrated into AEM's CSS, but as an example to share with others, I think this looks fine.

File: example-with-css.css (hunk 1 above)
Comment (OzzieOrca) [reply example-with-css.css:52]: I just picked a random color from Cru.org. I talked with Ivan after and he wants to do something like https://www.invisionapp.com/inside-design/ has with zooming images and text underlines of the same color (idk how important the animated underline is but sounds optional). I'll work on better hover states in a new PR.

**Comment by OzzieOrca** [comment]
Thanks for the review :)

File: example-with-css.css (hunk 1 above)
Comment (j2trumpet) [reply example-with-css.css:52]: @OzzieOrca That animated underline is a really creative use of background sizes. And that #5a5a5a color used to be the go-to dark gray tone, but it got replaced with #383F43 which you're using elsewhere.

**Pull request merged**